frontend/www/js/omegaup/ui.ts

```diff
@@ -68,7 +68,11 @@ export function formatString(
   });
 }
 
-export function displayStatus(message: string, type: string): void {
+export function displayStatus(
+  message: string,
+  type: string,
+  autoHide = true,
```

Comment:
```suggestion
  autoHide: boolean = true,
```

creo que si TypeScript se pone pedántico, como aquí únicamente se le está pasando la literal `true`, puede asumir que es el único tipo válido (onda, no sería un booleano: sólo puede recibir `true`). tons mejor seamos explícitos.

Igual en L117.

Reply:
Listo. Además aquí no se puede inicializar debido al comentario de mandar un objeto con todos los argumentos porque marca este error:
```
A type literal property cannot have an initializer.
```